client/src/components/Home.tsx

```diff
@@ -0,0 +1,11 @@
+import Navbar from "./layout/Navbar";
```

Comment:
maybe we could add this inside of a Home folder then expose it via a index.ts so it just does expert {default} from "./Home.tsx"?

Reply:
Yeah, we can